src/ralph/data_importer/management/commands/import_attachments.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+import csv
+import logging
+import os
+import tempfile
+import zipfile
+
+import tablib
+from django.apps import apps
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from ralph.attachments.helpers import add_attachment_from_disk
+from ralph.attachments.models import Attachment
+from ralph.back_office.models import BackOfficeAsset
+from ralph.data_center.models.physical import DataCenterAsset
+from ralph.data_importer.models import (
+    ImportedObjectDoesNotExist,
+    ImportedObjects
+)
+from ralph.lib.transitions.models import Transition, TransitionsHistory
+
+APP_MODELS = {model._meta.model_name: model for model in apps.get_models()}
+logger = logging.getLogger(__name__)
+
+MODELS_MAP = {
+    'BackOfficeAsset': BackOfficeAsset,
+    'DataCenterAsset': DataCenterAsset
+}
+
+
+class Command(BaseCommand):
+
+    help = "Imports attachments and Transition History"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'source',
+            help='zip source location',
+        )
+        parser.add_argument(
+            '-i', '--import',
+            choices=['attachments', 'transitionshistory', 'all'],
+            default='all',
+            dest='import'
+        )
+
+    def save_transition_history(self, directory):
+        transitions_map = {t.name: t for t in Transition.objects.all()}
+
+        with open(os.path.join(directory, 'transition_history.csv')) as f:
+            reader = csv.reader(f, delimiter=',')
+            csv_data = list(reader)
+            headers, csv_body = csv_data[0], csv_data[1:]
+            dataset = tablib.Dataset(*csv_body, headers=headers)
+            for line in dataset.dict:
+                model = MODELS_MAP.get(line.get('asset_type'))
+                try:
+                    object_imported_pk = ImportedObjects.get_new_id(
+                        model, line.get('asset')
+                    )
+                except ImportedObjectDoesNotExist:
+                    logger.warning(
+                        'Missing imported object for {}'.format(line)
+                    )
+                    continue
+
+                transition = transitions_map.get(line.get('transition'))
+                obj = model.objects.get(pk=object_imported_pk)
+                logged_user, _ = get_user_model().objects.get_or_create(
+                    username=line.get('logged_user')
+                )
+                history_imported_pk = None
+                try:
+                    history_imported_pk = ImportedObjects.get_new_id(
```

Comment:
TODO change to: https://github.com/allegro/ralph/pull/1949/files#diff-f7ac6f4977397d1d8734e408fdec8d9cR27